codespell_lib/_codespell.py

```diff
@@ -51,11 +51,11 @@
         False, None, supported_languages_en, None),
     ('rare', 'for rare but valid words', '_rare',
         None, None, None, None),
-    ('informal', 'for informal words', '_informal',
+    ('informal', 'for making informal words more formal', '_informal',
         True, True, supported_languages_en, supported_languages_en),
-    ('usage', 'for recommended terms', '_usage',
+    ('usage', 'for replacing phrasing with recommended terms', '_usage',
```

Comment:
@hadess does this seem an improved description of the usage dictionary or care to suggest further fine tuning?

Reply:
That does seem clearer to me, thanks.